$NetBSD: patch-ah,v 1.1 2006/01/12 00:48:36 joerg Exp $

Use the installed pkgsrc search-path file on OpenVMS, where imake is not
used to provide SYSPATHFILE.

--- path.c.orig	2006-01-12 00:44:46.000000000 +0000
+++ path.c
@@ -18,8 +18,10 @@
 #endif
 #include <errno.h>
 
-extern int errno;
-
+#if defined(VMS) && !defined(SYSPATHFILE)
+#define SYSPATHFILE "@PREFIX@/lib/X11/app-defaults/Xli"
+#endif
+
 static unsigned int NumPaths = 0;
 static unsigned int NumExts = 0;
 static char *Paths[BUFSIZ];
